<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="DESKTOP-C4EHALN" failures="3" tests="3" name="Default test(failed)(failed)" time="0.17" errors="0" timestamp="2024-07-26T21:13:55 IST">
  <testcase classname="testNg_Programs.NewTest1" name="login" time="0.024">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNg_Programs.NewTest1.login(NewTest1.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="testNg_Programs.NewTest1" name="logout" time="0.021">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNg_Programs.NewTest1.logout(NewTest1.java:46)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- logout -->
  <testcase classname="testNg_Programs.NewTest1" name="beforeClass" time="0.004">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNg_Programs.NewTest1.beforeClass(NewTest1.java:52)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- beforeClass -->
  <testcase classname="testNg_Programs.NewTest1" name="advsearch" time="0.0">
    <ignored/>
  </testcase> <!-- advsearch -->
  <testcase classname="testNg_Programs.NewTest1" name="search" time="0.0">
    <ignored/>
  </testcase> <!-- search -->
  <testcase classname="testNg_Programs.NewTest1" name="beforeTest" time="0.0">
    <ignored/>
  </testcase> <!-- beforeTest -->
  <testcase classname="testNg_Programs.NewTest1" name="afterTest" time="0.0">
    <ignored/>
  </testcase> <!-- afterTest -->
</testsuite> <!-- Default test(failed)(failed) -->
